opm/parser/eclipse/Parser/MessageContainer.cpp

```diff
@@ -128,6 +128,14 @@ namespace Opm {
     }
 
 
+    void MessageContainer::appendMessages(const MessageContainer& other)
```

Comment:
Why wouldn't you just use vector.insert? That way you allocate more efficiently.